### PR TITLE
Added simple documentation for bin sort algorithm

### LINKED ISSUE #442: Simple documentation for bin sort algorithm
# [Title] 
[short description - you can make links to any proper, publicly available explanations of the algorithm.]

# Implementations
[add links to currently available implementation under the corresponding folder according to the languages]

### PR DESCRIPTION
**Pull request by zinthanmm**
**Hi!**  
Thanks for considering contributing to this ever-growing list of algorithm and data structure implementations.
Your contribution is valuable. Use the follow template for creating Pull Requests
---

**Fixes #442 

By submitting this pull request I confirm I've read and complied with the below declarations.

-   [x] I have read the [Contribution guidelines](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/CONTRIBUTING.md) 
and I am confident that my PR reflects them.

-   [x] I have followed the [coding guidelines](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/CONTRIBUTING.md#cs) 
for this project.

-   [x] My code follows the [skeleton code structure](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/CONTRIBUTING.md#sample).

-   [x] This pull request has a descriptive title. 
For example, `Added {Algorithm/DS name} [{Language}]`, not `Update README.md` or `Added new code`.

-   [x] This pull request shall only be merged once the build passes.

-   [x] I have mentioned the issue number correctly (with hyper-link) in this pull request description.

### CODE REVIEW

File: algorithms/ar-binsrt/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+![alt text](https://cdn.programiz.com/sites/tutorial2program/files/Bucket_2.png)
+
+## Implementations
+
+Below are some examples of how the Bin Sort Algorithm is implemented using different programming languages.
+
+-   [Go](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/algorithms/ar-binsrt/golang/binsrt.go)
+-   [Java](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/algorithms/ar-binsrt/java/BinSort.java)
+-   [Python 3](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/algorithms/ar-binsrt/python3/binsrt.py)
```
Comment (RavinduSachintha) [inline algorithms/ar-binsrt/README.md:23]: Codacy found an issue: [[no-consecutive-blank-lines] Remove 1 line after node](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6318714)

**Review by RavinduSachintha — COMMENTED** [review]
@zinthanmm We cannot accept this contribution yet because as a guideline we keep one issue per PR rule. Here you have addressed several issues. Please remove unnecessary file changes.

**Review by zinthanmm — COMMENTED** [review]
I have done the necessary changes.

**Comment by RavinduSachintha** [comment]
> I have done the necessary changes.

there are some conflicts.

**Comment by zinthanmm** [comment]
I will send a new PR

**Pull request closed without merging**